fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/serializers/LookupDimensionToDimensionSpecSpec.groovy

```diff
@@ -67,6 +67,7 @@ class LookupDimensionToDimensionSpecSpec extends Specification{
     def "Given lookup dimension with no namespace serialize using dimension serializer"() {
         given:
         apiRequest.getDimensions() >> ([resources.d10])
```

Comment:
Can these be safely removed or not?  (applies throughout)

Reply:
No , it fails in constraints code with null pointer.
https://github.com/yahoo/fili/blob/master/fili-core/src/main/java/com/yahoo/bard/webservice/table/resolver/BaseDataSourceConstraint.java#L44